WireAPI/Sources/WireAPI/Extensions/JSONEncoder+Default.swift

```diff
@@ -27,7 +27,7 @@ extension JSONEncoder {
         encoder.outputFormatting = .sortedKeys
         encoder.dateEncodingStrategy = .custom({ date, encoder in
             var container = encoder.singleValueContainer()
-            let transportString = ISO8601DateFormatter.default.string(from: date)
+            let transportString = ISO8601DateFormatter.fractionalInternetDateTime.string(from: date)
```

Comment:
Is the backend actually able to handle both variants of timestamps, with and without fractional seconds, even if it sends different variants for certain fields?

Reply:
This I don't know, I think we can assume it's not, but current behavior is to always encode fractional seconds. In any case, I would open this discussion as a separate PR, if that's ok with you.

Reply:
Of course.